### PR TITLE
Added: Like and watch later functionality

### PR DESCRIPTION
**Pull request by hussiantabha**
User can now add, view & delete like videos. Also for watchlater.

### CODE REVIEW

File: src/pages/DisplayVideo.js [hunk 1]
```diff
@@ -21,100 +21,106 @@ const DisplayVideo = () => {
...
+          authorization: token,
+        },
+      });
+      const convertedJSON = await getData.json();
+      const filtered = await convertedJSON.playlists.filter(
+        (item) => item.title === playlist
+      );
+      const postData2 = await fetch(`/api/user/playlists/${filtered[0]._id}`, {
+        method: "POST",
+        headers: {
```
Comment (Kothalkarpriya) [inline src/pages/DisplayVideo.js:64]: Good that you used async await

**Review by Kothalkarpriya — APPROVED** [review]
Good work! You can merge

File: src/pages/DisplayVideo.js [hunk 2]
```diff
@@ -21,100 +21,106 @@ const DisplayVideo = () => {
...
-        authorization: token,
-      },
-    });
-    const convertedJSON = await getData.json();
-    dispatch({
-      type: "playlistData",
-      payload: { value: convertedJSON.playlists },
-    });
+    try {
+      const getData = await fetch("/api/user/playlists", {
```
Comment (t007rushi) [inline src/pages/DisplayVideo.js:25]: destructure here itself

File: src/pages/DisplayVideo.js [hunk 3]
```diff
@@ -125,6 +131,84 @@ const DisplayVideo = () => {
...
+          position: "top-right",
+          autoClose: 1000,
+          hideProgressBar: false,
+          closeOnClick: true,
+          pauseOnHover: false,
+          draggable: true,
+          progress: undefined,
+        });
+      }
+    } catch {}
```
Comment (t007rushi) [inline src/pages/DisplayVideo.js]: ```empty catch```

File: src/pages/DisplayVideo.js [hunk 4]
```diff
@@ -125,6 +131,84 @@ const DisplayVideo = () => {
...
+  };
+  //const likeArr = videoState.likedVideos.filter((item) => item.id === video.id);
```
Comment (t007rushi) [inline src/pages/DisplayVideo.js]: remove comments

**Review by t007rushi — APPROVED** [review]
Go ahead look for minor change suggetions

**Pull request merged**